src/rust/engine/src/externs/interface.rs

```diff
@@ -1014,6 +1014,7 @@ fn session_run_interactive_process(
       &Arc::new(std::sync::atomic::AtomicBool::new(true)),
       core.intrinsics.run(
         &Intrinsic {
+          id: RuleId::new("interactive_process"),
```

Comment:
This demonstrates that the lookup is now overspecified, and they key could just be the RuleId. But I left that for a followup, to keep this PR focused.